### PR TITLE
Brightpearl config from spree backend

### PR DESCRIPTION
**Pull request by stevenbarragan**
Set brightpearl config vars from spree backend

### CODE REVIEW

File: app/views/spree/admin/brightpearls/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<%= render 'spree/admin/shared/configuration_menu' %>
+
+<% content_for :page_title do %>
+  <%= Spree.t :brightpearl_settings  %>
+<% end %>
+
+<%= form_tag admin_brightpearl_path, :method => :put do %>
```
Comment (rodacato) [inline app/views/spree/admin/brightpearls/edit.html.erb]: dont use hash rockets if possible

**Comment by rodacato** [comment]
LGTM

**Comment by softr8** [comment]
LGTM

**Comment by vicmaster** [comment]
LGTM

**Pull request merged**